dumbui/index.html

```diff
@@ -135,9 +135,11 @@ <h3>KubeCon 2018: Object Storage -> Cloud Events -> Serverless</h3>
 
     // Pre-Authenticated Request URL for uploading into the "facedetection-incoming" bucket.
     // https://console.us-phoenix-1.oraclecloud.com/a/storage/objects/ocimiddleware/facedetection-incoming
+
     const sombreroSourceBucketURLBase = "https://objectstorage.us-phoenix-1.oraclecloud.com/p/svXDK-D4uP32oAq7jeewFg9oYfGimAZnWVmLIjBonA0/n/ocimiddleware/b/facedetection-incoming/o/";
```

Comment:
Hmm not sure about publishing the pre-baked PAR addresses for the opcmiddleware demo, maybe we need to find a way to make these configurable?  (or put a placeholder and tell the user to create a PAR and put it here)